### PR TITLE
[Skills] fixes for teams

### LINKED ISSUE #2339: RestaurantBooking's card is not clickable
#### What project is affected?

Skills

#### What language is this in?

C#

#### What happens?

When clicking RestaurantBooking's card, it goes wrong.

### PR DESCRIPTION
**Pull request by xieofxie**
### Purpose
*What is the context of this pull request? Why is it being done?*

Close #2339 

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

* remove ImageAssetLocation for Automotive, POI, RestaurantBooking, ToDo, Weather
  - IHttpContextAccessor is present in websocket
* add png for ToDo, Weather
  - Teams does not support svg now
* add 1.0 card, rename cusine to cuisine for RestaurantBooking
* format card jsons
* update docs

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

Y

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: skills/src/csharp/automotiveskill/automotiveskill/appsettings.json [hunk 1]
```diff
@@ -14,5 +14,7 @@
     "databaseId": "botstate-db",
     "cosmosDBEndpoint": "",
     "authKey": ""
-  }
+  },
+  // set this if used in skill mode
```
Comment (ryanisgrig) [inline skills/src/csharp/automotiveskill/automotiveskill/appsettings.json]: Remove the comments, but this optional setting should be noted in the documentation

File: skills/src/csharp/experimental/restaurantbooking/Dialogs/BookingDialog.cs [hunk 2]
```diff
@@ -145,7 +145,7 @@ private async Task<DialogTurnResult> AskForFoodType(WaterfallStepContext sc, Can
             foreach (var foodType in foodTypes)
             {
                 cards.Add(new Card(
-                    "CusineChoiceCard",
+                    GetDivergedCardName(sc.Context, "CusineChoiceCard"),
```
Comment (ryanisgrig) [inline skills/src/csharp/experimental/restaurantbooking/Dialogs/BookingDialog.cs]: Rename to CuisineChoiceCard

File: skills/src/csharp/experimental/restaurantbooking/RestaurantBooking.csproj [hunk 3]
```diff
@@ -22,6 +24,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="Content\CusineChoiceCard.1.0.json" />
```
Comment (ryanisgrig) [inline skills/src/csharp/experimental/restaurantbooking/RestaurantBooking.csproj]: Rename to CuisineChoiceCard

File: skills/src/csharp/experimental/restaurantbooking/RestaurantBooking.csproj [hunk 4]
```diff
@@ -7,6 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Remove="Content\CusineChoiceCard.1.0.json" />
```
Comment (ryanisgrig) [inline skills/src/csharp/experimental/restaurantbooking/RestaurantBooking.csproj]: Rename to CuisineChoiceCard

File: skills/src/csharp/experimental/weatherskill/appsettings.json [hunk 5]
```diff
@@ -20,5 +20,7 @@
     "cosmosDBEndpoint": "",
     "databaseId": ""
   },
+  // set this if used in skill mode
```
Comment (ryanisgrig) [inline skills/src/csharp/experimental/weatherskill/appsettings.json]: Remove comments

File: skills/src/csharp/todoskill/todoskill/appsettings.json [hunk 6]
```diff
@@ -25,6 +25,8 @@
     "cosmosDBEndpoint": "",
     "authKey": ""
   },
+  // set this if used in skill mode
```
Comment (ryanisgrig) [inline skills/src/csharp/todoskill/todoskill/appsettings.json]: Remove comments

**Review by ryanisgrig — CHANGES_REQUESTED** [review]
Opening an issue for the docs changes

**Comment by xieofxie** [comment]
After some testing, I found that IHttpContextAccessor is present in websocket, so I removed ImageAssetLocation for Automotive, POI, RestaurantBooking, ToDo, Weather and related documents.

I also found that Teams does not support svg now (how?), so pngs are added for ToDo, Weather.

Rename all cusine to cuisine for RestaurantBooking.

**Review by darrenj — APPROVED** [review]

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**